url.bs

```diff
@@ -85,6 +85,170 @@ valid input. User agents, especially conformance checkers, are encouraged to rep
+   <td>This error occurs when a URL with a <a>special scheme</a> uses the U+005C (\) reverse
+   solidus code point instead of the U+002F (/) code point.
+  <tr>
+   <td><dfn>missing-solidus-before-authority</dfn>
+   <td>This error occurs if the URL <a>includes credentials</a> and they are not preceeded by
+   two "<code>//</code>" U+002F (/) code points.
+  <tr>
+   <td><dfn>unexpected-@-code-point</dfn>
+   <td>This error occurs when the URL <a>includes credentials</a>, as the authority porition of
+   a URL is considered invalid.
+  <tr>
+   <td><dfn>missing-credentials</dfn>
+   <td>This error occurs when an U+0040 (@) code point is found, but the URL does not
+   <a>include credentials</a>.
+  <tr>
+   <td><dfn>unexpected-port-without-host</dfn>
+   <td>This error occurs when the URL contains a <a>port</a>, but no <a for=url>host</a>.
+  <tr>
+   <td><dfn>empty-host-special-scheme</dfn>
+   <td>This error occurs when a URL with a <a>special scheme</a> does not contain a
+   <a for=url>host</a>.
+  <tr>
+   <td><dfn>invalid-host</dfn>
+   <td>This error occurs when the host portion of a URL is set to the empty string when the URL
+   <a>includes credentials</a> or a <a>port</a> and the <a>basic URL parser</a>'s state is
+   overridden. (e.g. When setting {{URL}}'s {{URL/host}} attribute to an empty string.)
+  <tr>
+   <td><dfn>port-out-of-range</dfn>
+   <td>This error occurs when the <a>port</a> is a valid number, but exceeds 65,535.
+  <tr>
+   <td><dfn>invalid-port</dfn>
+   <td>This error occurs when a non-integer value is found when parsing the <a>port</a>.
+  <tr>
+   <td><dfn>unexpected-windows-drive-letter</dfn>
+   <td>This error occurs when a <a>relative-URL string</a>
+   <a>starts with a Windows drive letter</a> and the <a>base URL</a>'s <a>scheme</a> is
+   "<code>file</code>".
+  <tr>
+   <td><dfn>unexpected-windows-drive-letter-host</dfn>
+   <td>???
+  <tr>
+   <td><dfn>unexpected-host-file-scheme</dfn>
+   <td>This error occurs when the URL's <a>scheme</a> is changed to the "<code>file</code>"
+   <a>scheme</a> and the existing URL has a <a for=url>host</a>.
+  <tr>
+   <td><dfn>unexpected-empty-path</dfn>
+   <td>This error occurs when a URL with a "<code>file</code>" <a>scheme</a> contains an empty
+   <a>path</a>.
+  <tr>
+   <td><dfn>invalid-url-code-point</dfn>
+   <td>This error occurs when a code point that is not a <a>url code point</a> or U+0025 (%) in the
+   URL's <a>path</a>, <a>query</a>, or <a for=url>fragment</a>.
+  <tr>
+   <td><dfn>unescaped-percent-sign</dfn>
+   <td>This error occurs when a U+0025 (%) code point is found that is not followed by two
+   <a>ASCII hex digits</a>, representing a <a>percent-encoded byte</a>, in the URL's <a>path</a>,
+   <a>query</a>, or <a for=url>fragment</a>.
+  <tr>
+   <td><dfn>unclosed-ipv6-address</dfn>
+   <td>This error occurs when an IPv6 address is missing the closing U+005D (]) code point.
+  <tr>
+   <td><dfn>ascii-idn-failure</dfn>
```

Comment:
There are more errors here now so you'll have to split this (and rebase). I suggest calling this after IDNA, ToASCII, or domain-to-ASCII and not add a new term (IDN).